lyricsgenius/get-album.py

```diff
@@ -0,0 +1,66 @@
+import pandas as pd
+from bs4 import BeautifulSoup
+import requests
+import lyricsgenius as genius
+
+client_access_token = 'CLIENTACCESSTOKEN'
+
+
+def search_album(artist_name, album_title):
+    api = genius.Genius(client_access_token)
+    artist = api.search_artist(artist_name, max_songs=0)
+    artist_name = artist_name.replace(" ", "-").replace(".", "").replace("$", "-").replace("\"", "").replace("'", "")
+    album_title = album_title.replace(" ", "-").replace(".", "-").replace("$", "-").replace("'", "-").replace("\"", "-")
+    while artist_name[-1] == '-':
+        artist_name = artist_name[:-1]
+    while album_title[-1] == '-':
+        album_title = album_title[:-1]
+    index = []
+    r = requests.get('https://genius.com/albums/' + artist_name + "/" + album_title)
```

Comment:
I understand what you're doing here, but I'm worried this approach of replacing spaces and formatting for the `genius.com/albums/artist_name/album_title` URL would be prone to producing URLs that aren't really on Genius.com.

Reply:
Yeah it definitely seems risky but so far, it worked. I'm going to put it to the test a lot more to be more sure.